09.mtools.1.dpatch by A. Costa <agcosta@gis.net>
'man mtools' typos #468744

--- mtools-3.9.11/mtools.1	2007-04-11 23:16:01.000000000 +1000
+++ mtools-3.9.11/mtools.1	2008-04-08 21:02:10.000000000 +1000
@@ -458,12 +458,12 @@ data than the others. Unfortunately, DOS
 each track contains the same amount of data. Thus 2m hides the fact that
 the first track contains less data by using a \fIshadow
 FAT\fR. (Usually, DOS stores the FAT in two identical copies, for
-additional safety.  XDF stores only one copy, and it tells DOS that it
-stores two. Thus the same that would be taken up by the second FAT copy
-is saved.) This also means that your should \fBnever use a 2m disk
+additional safety.  XDF stores only one copy, but tells DOS that it
+stores two. Thus the space that would be taken up by the second FAT copy
+is saved.) This also means that you should \fBnever use a 2m disk
 to store anything else than a DOS fs\fR.
 .PP
-Mtools supports these format only on Linux.
+Mtools supports these formats only on Linux.
 .PP
 .SS \ \ XDF
 .iX "c XDF disks"
